ui/app/AppLayouts/Chat/stores/UsersStore.qml

```diff
@@ -1,12 +1,12 @@
-import QtQuick 2.13
+import QtQuick 2.15
 
 QtObject {
     id: root
 
     property var usersModule
 
-    readonly property var usersModel: usersModule ? usersModule.model : null
-    readonly property var temporaryModel: usersModule ? usersModule.temporaryModel : null
+    property var usersModel: usersModule ? usersModule.model : null
```

Comment:
@micieslak this has to be writable

Reply:
Hmm, why?

Reply:
See the hunk below in CreateChatView; the `usersModel` gets overridden